helpers/index.js

```diff
@@ -11,6 +12,10 @@
    * var app = express();
    * app.use(helpers.errorHandler);
    * */
+
+  var gauge = new client.Gauge('nodejs_errors_total', 'This is a counter of all the exceptions handled.');
```

Comment:
Guage is probably not what you want here. You'd need a counter to count things. A guage can be set, a counter cant.